src/runtime/node/crypto/web.ts

```diff
@@ -5,10 +5,16 @@ import type nodeCrypto from "node:crypto";
 export const CryptoKey =
   globalThis.CryptoKey as unknown as typeof nodeCrypto.webcrypto.CryptoKey;
 
-export const webcrypto: Crypto & typeof nodeCrypto.webcrypto = {
-  CryptoKey,
```

Comment:
Maybe we can simply bind CryptoKey in L5 to avoid proxy?

Reply:
I haven't tried that, and I don't think I fully understand what you mean to be honest. Can you show me a code example so I can try locally?

Reply:
`const CryptoKey = globalThis.CryptoKey.bind(globalThis.CryptoKey)`. Please try it on your repo (if you have a reproduction that would be amazing too!)

Reply:
I'm using Nuxt with Deno Deploy, so the issue is somewhat nested in the dependency chain, but I'll try to create a minimal sample.

Reply:
https://github.com/AaronDewes/unenv-bug-demo There's also some more explanation of the bug in this repo. I hope this shows the issue well enough. Nitro is probably not needed, but was the easiest for me to set up as a demo.

Reply:
Thank you for nice reproduction! Have made some few refactors to make sure node webcrypto is not affecting us also simplified proxy (by using main crypto as target, proxy handles calls with the right context)